cpp/CMakeLists.txt

```diff
@@ -446,6 +446,7 @@ add_library(
   src/join/mixed_join_size_kernel_nulls.cu
   src/join/mixed_join_size_kernels_semi.cu
   src/join/semi_join.cu
+  src/join/distinct_hash_join.cu
```

Comment:
This was renamed but it's no longer in alphabetical order. Move just below `cross_join.cu`.

Reply:
Done 119a5312e7e364fcdcd057f846dfa9b5b6df3aef